docs/content/platform/blocks/exa/search.md

```
@@ -0,0 +1,49 @@
+# Exa Search
+
+### What it is
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)